x-pack/plugins/infra/common/snapshot_metric_i18n.ts

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
Note to self: switch to the implementation in https://github.com/elastic/kibana/pull/69757 instead